test/correctness/fuse.cpp

```diff
@@ -72,7 +72,7 @@ int main(int argc, char **argv) {
         Var xy("xy");
         f.compute_root()
             .fuse(x, y, xy)
-            .vectorize(xy, 16);
+            .vectorize(xy, 16, TailStrategy::RoundUp);
```

Comment:
Is this change related to the PR?

Reply:
Yes, the stronger simplification injected a new mod operation, which wasn't the mod operation we were trying to assert didn't exist.